cmd/guard.go

```diff
@@ -0,0 +1,83 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/hashicorp/go-version"
+	"github.com/leancodepl/poe2arb/flutter"
+	"github.com/spf13/cobra"
+)
+
+type flutterConfigKey int
+
+const key flutterConfigKey = 1
+
+func contextWithFlutterConfig(ctx context.Context, flutterConfig *flutter.FlutterConfig) context.Context {
```

Comment:
We can keep it in `getFlutterConfigAndEnsureSufficientVersion`, it won't be used elsewhere